config/groovy/util.groovy

```diff
@@ -203,42 +266,47 @@ private void printListItemsCondensed(String[] items) {
-    println ""
+
+    println("""
+    Utility script for interacting with Destination Sol. General syntax:
+      groovyw (type) (sub-command)
+    - 'type' may be module or lib (Destination Sol still only supports a subset vs Terasology)
+    
+    Available sub-commands:
+    - 'get' - retrieves one or more items in source form (separate with spaces)
+    - 'get-all' - retrieves all modules that can be found on the configured remote locations
+    - 'recurse' - retrieves the given item(s) *and* their dependencies in source form (really only for modules)
+    - 'list' - lists items that are available for download or downloaded already.
+    - 'create' - creates a new item of the given type.
+    - 'update' - updates an item (git pulls latest from current origin, if workspace is clean
+    - 'update-all' - updates all local items of the given type.
+    - 'add-remote (item) (name)' - adds a remote (name) to (item) with the default URL.
+    - 'add-remote (item) (name) (URL)' - adds a remote with the given URL
+    - 'list-remotes (item)' - lists all remotes for (item) 
+    - 'refresh' - replaces the Gradle build file for all items of the given type from the latest template
+    - 'createDependencyDotFile' - creates a dot file recursively listing dependencies of given locally available module, can be visualized with e.g. graphviz
+    
+    Available flags:
+    '-remote [someRemote]' to clone from an alternative remote, also adding the upstream org (like MovingBlocks) repo as 'origin'
+           Note: 'get' + 'recurse' only. This will override an alternativeGithubHome set via gradle.properties.
+    '-simple-list-format' to print one item per row for the 'list' sub-command, even for large numbers of items
+    '-condensed-list-format' to group items by starting letter for the 'list' sub-command (default with many items)
+    '-skip-recently-updated' (Only for update-all) to skip updating modules that have already been updated within 10 minutes
+    
+    Example: 'groovyw module create MySpaceShips' - would create that module
+    Example: 'groovyw module get caution - remote vampcat - would retrieve caution module from vampcat's account on github.'
+    Example: 'groovyw module get *' - would retrieve all the modules in the DestinationSol organisation on GitHub.
+    Example: 'groovyw module get ?r*' - would retrieve all the modules in the DestinationSol organisation on GitHub
+        that start with any single character, then an \\"r\\" and then end with anything else.
+        This should retrieve the draconic, organic and tribe repositories from the DestinationSol organisation on GitHub.
+ 
+    *NOTE*: On UNIX platforms (MacOS and Linux), the wildcard arguments must be escaped with single quotes e.g. groovyw module get '*'.
+    
+    *NOTE*: Item names are case sensitive. If you add items then `gradlew idea` or similar may be needed to refresh your IDE
```

Comment:
I don't actually use `gradlew idea` at all anymore, since IntelliJ's built-in gradle support just works now. Is this still standard practice for Destination Sol?

Reply:
Nah, pretty sure this is just obsolete and we never got around to removing it. There are still stray references to `gradlew idea` in some of the wikis too, I think.